src/test/java/integration/BaseTest.java

```diff
@@ -36,6 +36,9 @@ public class BaseTest {
     public static void setUp() {
         spKey = System.getenv("SERVICE_PRINCIPAL_KEY");
         String accessKeyBase64 = System.getenv("ACCESS_KEY");
+        if (!nullOrEmpty(accessKeyBase64)) {
```

Comment:
before this, if config is loaded from environment (instead of .env file), then accessKey was not initialized, causing the test not being executed.